pkg/pipeline/transform/transform_network.go

```diff
@@ -95,6 +95,18 @@ func (n *Network) TransformEntry(inputEntry config.GenericMap) config.GenericMap
 				outputEntry[rule.Output] = outputEntry[rule.Input]
 				outputEntry[rule.Output+"_Evaluate"] = true
 			}
+		case api.TransformNetworkOperationName("AssignIf"):
+			expressionString := fmt.Sprintf("val %s", rule.Parameters)
```

Comment:
Eran asked to combine the `AssignIf` into the existing `AddIf`. The code is essentially the same, with the exception of what gets assigned to the the Output variable. If an assigned value is provided, it gets the new value; otherwise it gets the value of the Input variable.